masterlist.yaml

```
@@ -17471,13 +17471,15 @@ plugins:
         itm: 122
 
   # Additional Clockwork
+  - name: 'Clockwork.+?\.esp'
```

The quantifier doesn't need to be made lazy in this case:

```suggestion
  - name: 'Clockwork.+\.esp'
```